root/types.js

```diff
@@ -416,10 +416,12 @@ declare type EditT = {|
   |},
   +created_time: string,
   +data: Object,
+  +display_data: Object,
```

Comment:
Hmm. This specific one has $data->{$entity_type} = $loaded->{$model}->{$self->$entity_id} so it has work: WorkT if it's a work, artist: ArtistT for an artist, etc. How can we indicate this?

Reply:
Good question...there's an `AnnotatedEntityT` in root/static/scripts/common/components/Annotation.js so maybe we can move that to root/types.js and make use of it.

This might work. Technically it would allow things like `artist: WorkT` but it doesn't really matter since it's a generic component and the props come from the server anyway (meaning, if the server screws up the data, better Flow types don't help us).

```JavaScript
type AnnotatedEntityTypeT = $ElementType<AnnotatedEntityT, 'entityType'>;

// ...
+display_data: {|
  +entity_type: AnnotatedEntityTypeT,
  [AnnotatedEntityTypeT]: AnnotatedEntityT,
|},
```

Reply:
https://github.com/metabrainz/musicbrainz-server/pull/881